Formula/cairo.rb

```diff
@@ -17,6 +17,13 @@ class Cairo < Formula
     sha256 "268cc265a7f807403582f440643064bf52896556766890c8df7bad02d230f6c9"
   end
 
+  head do
+    url "http://anongit.freedesktop.org/git/cairo", :using => :git
```

Comment:
Er, `:using => :git` even. We don't offer support for fetching URLs with one's gut 😅.